cves/CVE-2008-0005.yml

```diff
@@ -202,8 +206,8 @@ lessons:
     applies:
     note:
   complex_inputs:
```

Comment:
If the problem was an input of UTF-7 when there was no encoding specified, could this also be considered distrust input. It seems like the browser is assume that the input can be auto assigned without any sanitization.

Reply:
Good catch